framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -91,6 +91,10 @@ namespace bpl {
 #define DEFAULT_BAND_STEERING 1
 #define DEFAULT_DFS_REENTRY 1
 #define DEFAULT_CLIENT_ROAMING 1
+#define DEFAULT_PERSISTENT_DB 0
+#define DEFAULT_CLIENTS_PERSISTENT_DB_MAX_SIZE 256
```

Comment:
I think it is 255 clients. This is what we test on Veriview.

Reply:
this is not the number of clients currently connected - but the number of clients which their data is currently stored in the persistent DB.
they might all not be connected at the moment.
and it was agreed that we set its default to 256

Reply:
I meant that the max allowed clients is 255. Because that what we test on Veriview.

Reply:
the max-allowed-clients set as 255 affect the currently-connected clients - not the number of clients that we can cache in the persistent DB.
see my other comment: https://github.com/prplfoundation/prplMesh/pull/1505#discussion_r449840480